lib/routes/autocentre/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Data, Route } from '@/types';
+import parser from '@/utils/rss-parser';
+
+export const route: Route = {
+    path: '/',
+    name: 'Автомобільний сайт N1 в Україні',
+    categories: ['new-media'],
+    maintainers: ['driversti'],
+    example: '/autocentre',
+    handler,
+};
+
+const createItem = (item) => ({
+    title: item.title,
+    link: item.link,
+    description: item.contentSnippet,
+});
+
+async function handler(): Promise<Data> {
+    const feed = await parser.parseURL('https://autocentre.ua/rss');
```

Comment:
You can use the redirected URL
```suggestion
    const feed = await parser.parseURL('https://www.autocentre.ua/rss');
```